src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -21,7 +21,7 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
     const styles = useThemeStyles();
     const policyID = policy?.id ?? '';
     const {exportCompanyCardAccount, exportAccountPayable, autoCreateVendor, errorFields, pendingFields, exportCompanyCard} = policy?.connections?.quickbooksOnline?.config ?? {};
```

Comment:
NAB: I don't think `exportCompanyCardAccount`,  `exportAccountPayable` or `exportCompanyCard` really exists in the backend

Reply:
these were also added by Nikolay in [another PR](https://github.com/Expensify/App/pull/40902) 😨